pgcli/pgexecute.py

```diff
@@ -24,6 +26,10 @@
 # See http://initd.org/psycopg/articles/2014/07/20/cancelling-postgresql-statements-python/
 ext.set_wait_callback(psycopg2.extras.wait_select)
 
+FunctionMetadata = namedtuple('FunctionMetadata',
+                              ['schema_name', 'func_name', 'arg_list', 'result',
```

Comment:
Yes, it's the literal `returns` clause as specified in the `create function` definition. `return_type` is a better name than `result`